src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,59 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    parser.add_argument('--arch', dest='arch', type=str, help='Architecture', default='x86_64')
+    args = parser.parse_args()
+
+    builds = getBuildsForStream(args.stream, args.arch)
+    for build in builds:
+        print("The build is "+build)
+        buildFetch(args.stream, build, args.arch)
+        meta = open('builds/'+build+'/'+args.arch+'/meta.json')
+        data = json.load(meta)
+
+        # Delete this when actually running. Just here while I make this script
+        # data ={"amis":[{
+        #     "name": "us-east-1",
+        #     "hvm": "ami-0016d5df3041499f9",
+        #     "snapshot": "snap-0c1ca4850fcd5e573"
+        # }]}
+        amis = data['amis']
+        for ami in amis:
+            UpdateTagCmd = 'aws ec2 create-tags --resource ' + ami['hvm'] + ' --tags '+ 'Key="FedoraUser",Value="coreos"' + ' --region=' + ami['name']

```

Comment:
another instance where f strings might make things easier to read:

```suggestion
            UpdateTagCmd = f'aws ec2 create-tags --resource {ami["hvm"]} --tags Key="FedoraUser",Value="coreos" --region {ami["name"]}

```

Reply:
double check also: this should be `FedoraGroup` right, not `FedoraUser`?